airlock/renderers.py

```diff
@@ -40,7 +40,9 @@ class Renderer:
     def get_response(self):
         if self.template:
             context = self.context()
-            response = SimpleTemplateResponse(self.template.template, context)
+            response: HttpResponseBase = SimpleTemplateResponse(
+                self.template.template, context
+            )
         else:
             response = FileResponse(self.abspath.open("rb"), filename=self.filename)
```

Comment:
Why don't we need to do `response: HttpResponseBase = FileResponse(...)` here?  Is it because mypy's picked up the type from the first branch?

Reply:
Yes, in general mypy wants variables to have a single type. Without the explicit type assignment it assumes that `response` has type `SimpleTemplateResponse` which is incompatible with `FileResponse`. So we have to explicitly give it a wider type here. We could also be explicit within this branch but that would be redundant.